src/when-you-get-an-alert.html

```diff
@@ -53,6 +53,9 @@ <h1 class="govuk-heading-xl">
           read out the alert
         </li>
       </ul>
+      <p class="govuk-body">
+        The sound and vibration will last for 10 seconds.
```

Comment:
How do we know this?

Reply:
I checked this with Pete D at Fujitsu.

After reviewing the video from the Sizewell test, he confirmed that all phones stopped making the sound after approx 10 seconds.

I’ll add ‘about’ to ~remove~ _include_ a little ambiguity. ;-)

Reply:
```suggestion
        The sound and vibration will last for about 10 seconds.
```